dev-docs/tls-cookbook.md

```diff
@@ -121,6 +122,10 @@ kubectl create secret generic \
 3. Update all the service certificates:
 
 ```
+export HAIL=$HOME/hail
```

Comment:
Let's remove this, in dev-docs/hail-overview.md we state that $HAIL is the root of the hail repo. I think we can assume they've seen that before they come here. Some folks (myself) keep the hail repository elsewhere.

Reply:
Sure, done!